.gitignore

```diff
@@ -1,4 +1,12 @@
+# Test coverage
 coverage
+
+# Library files
 node_modules
+
+# Log files
 npm-debug.log
+
+# Read this 👇 for putting package-lock in gitignore
```

Comment:
Can you remove all this boiler plate? It makes it difficult to maintain an entire org of modules if suddenly things like `.gitignore` now differ between each repo.